app/src/main/java/edu/gatech/cs2340/hkskh/Users/Models/User.java

```diff
@@ -82,5 +82,29 @@ public void setNumBeds(int beds, boolean family, boolean release) {
             }
         }
     }
+
+    /**
+     * want to display amount of beds claimed
+     * @return number of family spots claimed
+     */
+    public int getNumFamily() {
+        return this.numFamily;
+    }
+
+    /**
+     * want to display the number of individual spots claimed
+     * @return number of individual spots claimed
+     */
+    public int getNumInd() {
+        return this.numInd;
+    }
```

Comment:
These are most likely going to cause merge conflicts with my branch since I added them as well. I can add `isCheckedIn` to my code.